### PR TITLE
Fix upgrade scenario

### PR DESCRIPTION
**Pull request by imobach**
* This PR fixes _upgrade scenario_ for both providers (VirtualBox and libvirt).
* BTW, it sets `VAGRANT_LOG` to `warn`.

### CODE REVIEW

File: lib/ay_tests/image_builder.rb [hunk 1]
```diff
@@ -210,7 +237,22 @@ def change_boot_order
     # The image will be destroyed be Veewee when starting the upgrade. So we need to
```
Comment (schubi2) [inline lib/ay_tests/image_builder.rb]: A little typo :-)

File: kiwi/definitions/autoyast/upgrade_definition.rb [hunk 2]
```diff
@@ -46,11 +46,22 @@
...
-      puts "generating autoyast image with mac address: #{mac}"
-      system "sudo virt-clone -o autoyast_sav -n autoyast --file /var/lib/libvirt/images/autoyast.qcow2 --mac #{mac}"
-      system "sudo virsh undefine autoyast_sav --remove-all-storage"
+      # FIXME: it's a little bit tricky and it deserves some refactoring/cleanup.
+      if ENV["AYTESTS_PROVIDER"] == "virtualbox"
+        vm_config = `VBoxManage showvminfo autoyast | grep "Config file" | cut -f2 -d:`.strip
+        vm_dir = File.dirname(vm_config)
+        system "VBoxManage unregistervm autoyast --delete"
+        FileUtils.mv vm_dir.sub("autoyast", "autoyast.sav"), vm_dir
+        system "VBoxManage registervm \"#{vm_config}\""
```
Comment (schubi2) [inline kiwi/definitions/autoyast/upgrade_definition.rb]: Could you please add here some "puts" in order to show what is going on ?

Comment (imobach) [inline kiwi/definitions/autoyast/upgrade_definition.rb]: Yes, you're right. But Veewee won't have access to log.info at this point (as it's an external program). Anyway, it's something we should improve in the near future (maybe defining the place where log messages should go using a environment variable).

**Comment by schubi2** [comment]
LGTM

**Comment by imobach** [comment]
@schubi2 Thanks. Merging...

**Pull request merged**